src/services/edusharing/hooks/index.js

```diff
@@ -0,0 +1,31 @@
+const { Configuration } = require('@schul-cloud/commons');
+const { authenticate } = require('@feathersjs/authentication');
+const { disallow } = require('feathers-hooks-common');
+const { Forbidden } = require('@feathersjs/errors');
+
+const isEdusharing = (context) => {
+	if (Configuration.get('LERNSTORE_MODE') !== 'EDUSHARING') {
+		throw new Forbidden('This API is activated only for the lernstore mode Edusharing');
```

Comment:
Why Forbidden? Shouldn't this be something like NotFound or NotImplemented instead?

Reply:
Ok Notfound.